### PR TITLE
Updating workshop request form

### PR DESCRIPTION
**Pull request by gvwilson**

### CODE REVIEW

File: workshops/request.html [hunk 1]
```diff
@@ -4,8 +4,37 @@
...
+    For more information on how our workshops operate,
+    please read our <a href="{{page.root}}/workshops/operations.html">Operations Guide</a>.
+  </p>
+  <p>
+    Our instructors are volunteers,
+    and so are not paid for their teaching,
+    but host sites are required to cover their travel and accommodation costs.
+    In addition,
+    the Software Carpentry Foundation
+    <a href="{{page.root}}/scf/workshops.html">charges an administration fee</a>
```
Comment (katyhuff) [inline workshops/request.html]: I wonder if this link should actually be pointing here: http://software-carpentry.org/workshops/operations.html#host_cost ?

File: workshops/request.html [hunk 2]
```diff
@@ -4,8 +4,37 @@
...
+    for workshops that it helps organize at institutions which are not
+    <a href="{{page.root}}/scf/membership.html">partners or affiliates</a>.
+    This is presently US$1250 for the first worskhop at an institution in any given year,
+    and US$750 for each subsequent workshop in that year at that institution.
```
Comment (katyhuff) [inline workshops/request.html]: Right here you need a sentence that says how to pay the fee. "To pay the fee, use paypal" or somesuch.

**Comment by katyhuff** [comment]
looks good. You've definitely made it clearer that folks have to pay the fee. I added two comments targeted at helping folks figure out *how* to pay the fee.

File: workshops/request.html (hunk 1 above)
Comment (katyhuff) [inline workshops/request.html]: And, actually, this is tricky... neither link actually says HOW to pay the fee. Both of them should say how to pay. Honestly, if you asked me tomorrow to pay a workshop fee, I, Katy Huff, would have to send you an email to find out how to do it. It's not clear at all from the website where the fees should be paid and who needs to be informed.

**Comment by r-gaia-cs** [comment]
:+1: to merge.

**Comment by JasonJWilliamsNY** [comment]
I wonder if we also comment that the workshop fee is due paid in full prior to the start of the workshop? That would encourage the host to have their payment mechanism in place of the date the choose to have it hosted.

**Comment by gvwilson** [comment]
We've never required pre-payment in the past.  What do we do if they 
don't come through - cancel the workshop?

**Comment by r-gaia-cs** [comment]
> We've never required pre-payment in the past.

Maybe require payment before the announcement in our site?

**Comment by gvwilson** [comment]
I think we should wait until the workshop takes place before invoicing - 
we've had to cancel at the last minute more than once, and refunding is 
a pain.

**Comment by r-gaia-cs** [comment]
> I think we should wait until the workshop takes place before invoicing - 
> we've had to cancel at the last minute more than once, and refunding is 
> a pain.

OK.

**Comment by katyhuff** [comment]
Wonderful!! Looks great to me!

**Comment by karinlag** [comment]
Also looks good to me.

**Comment by JasonJWilliamsNY** [comment]
The 'pay before workshop' comment was a thought on applying pressure to get payments to us in a timely manner. It may introduce problems due to cancellations as you say (although now with NumFocus I perhaps refunds may be easier?). Another alternative proposition would be having fines as disincentives for not paying within 60 days.

**Comment by gvwilson** [comment]
I don't know how we'd enforce a fine, but if the Steering Committee votes for it, sure.  Merge this as-is and put it on the agenda?

**Comment by JasonJWilliamsNY** [comment]
Sounds good. I think the finance committee should get to your input on how bad the problem is (or using) and propose some solutions to the SC. If this problem is more than folks just being misinformed about our fee, having some 'penalty' even if enforcement is not difficult may motivate compliance. Commercial vendors who supply universities have reasonable expectations of how long it takes for payments, but they don't tolerate weeks long back-and-forth. Perhaps with NumFocus we have new options: having hosts pay by university credit cards should be very quick.

**Comment by r-gaia-cs** [comment]
This looks ready to merge for me.

**Pull request merged**